.github/workflows/ci.yml

```
@@ -43,7 +43,7 @@ jobs:
         run: dotnet build -c Release --no-restore
 
       - name: Test
-        run: dotnet test -c Release --no-build --logger GitHubActions
+        run: dotnet test -c Release --no-build --logger "console;verbosity=detailed"
```

```suggestion
        run: dotnet test --blame -c Release --no-build --logger "console;verbosity=detailed"
```
@kylejuliandev months ago, I came across this "tweet": https://bsky.app/profile/davidfowl.com/post/3lflw4apqgc2w
Might worth a try